### PR TITLE
API Updates 

### PR DESCRIPTION
**Pull request by Armando-Ibarra**
Added API key auth to Integrated Platforms Balances (CS2)
Added support for Trustly & Cv Connect APM (CS2)
Review payment & payout request schema
Added patch endpoint for payment instruments (CS2)

### CODE REVIEW

**Review by armandojaleo — APPROVED** [review]
👍

File: lib/Checkout/Payments/ProcessingSettings.php [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 namespace Checkout\Payments;
 
+use Checkout\Payments\Previous\SenderInformation;
+
 class ProcessingSettings
```
[the remark this answers is not in the document]
Comment (martinseco) [reply lib/Checkout/Payments/ProcessingSettings.php:7]: [Api reference](https://api-reference.checkout.com/previous/#operation/requestAPaymentOrPayout!path=1/processing/purpose&t=request)

File: lib/Checkout/Payments/PaymentRecipient.php [hunk 2]
```diff
@@ -14,6 +16,11 @@ class PaymentRecipient
...
     public $account_number;
 
+    /**
+     * @var Address
+     */
+    public $address;
+
     /**
      * @var string
      */
```
Comment (martinseco) [inline lib/Checkout/Payments/PaymentRecipient.php:26]: `Zip` is deprecated, maybe we should add a comment here

Comment (Armando-Ibarra) [reply lib/Checkout/Payments/PaymentRecipient.php:26]: It's deprecated for NAS but not for MBC, that's why I didn't add the deprecated annotation

**Pull request merged**